### PR TITLE
Revert to Ubuntu 18.04

### PR DESCRIPTION
**Pull request by lukicdarkoo**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 name: webots
 title: Webots
-base: core20
+base: core18
 version: 'R2021a'
```
Comment (stefaniapedrazzi) [inline snapcraft.yaml]: Note that in the final version we should also change this line
```suggestion
version: 'R2021b'
```

**Comment by lukicdarkoo** [comment]
@stefaniapedrazzi This runs, but then we have the old problem, it doesn't work with ROS Noetic.

**Comment by omichel** [comment]
I would suggest to stay on Melodic for the snap. Noetic users can use the tarball or the deb versions.

**Comment by stefaniapedrazzi** [comment]
As far as I understand about the issue on `noetic` it seems it is just a matter of setting the LD_LIBRARY_PATH to make it work with `noetic`. Isn't it?
So if we document it users will also be able to use the snap package.

**Comment by lukicdarkoo** [comment]
> As far as I understand about the issue on `noetic` it seems it is just a matter of setting the LD_LIBRARY_PATH to make it work with `noetic`. Isn't it?
> So if we document it users will also be able to use the snap package.

Yes, we can document it

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**